src/app/ui/landing-page/landing-page.js

```diff
@@ -40,17 +40,50 @@ let css = csjs`
     width           : 100%;
     font-size       : xx-large;
     align-self      : center;
-
   }
   .text {
     cursor: pointer;
     font-weight: normal;
     max-width: 300px;
     user-select: none;
+    color: var(--primary);
```

Comment:
I believe we should stick to light/dark because light and primary can be quite close and the text would be not readable